pkg/crd/crds.go

```diff
@@ -43,7 +43,10 @@ import (
 )
 
 // CRDs to apply
-var crdNames = map[string]bool{"placements.edge.kubestellar.io": true}
+var crdNames = map[string]bool{
```

Comment:
What is the `bool` for?

Why not use https://github.com/kubernetes/apimachinery/blob/v0.28.2/pkg/util/sets/set.go#L31 ?